src/libaktualizr/package_manager/packagemanagerfake_test.cc

```diff
@@ -26,7 +26,8 @@ TEST(PackageManagerFake, FinalizeAfterReboot) {
 
   PackageManagerFake fakepm(config.pacman, storage, bootloader, nullptr);
 
-  Uptane::Target target("pkg", {Uptane::Hash(Uptane::Hash::Type::kSha256, "hash")}, 1, "");
+  std::map<Uptane::EcuSerial, Uptane::HardwareIdentifier> primary_ecu;
```

Comment:
It's not exactly related to this specific PR rather to the code base in general - this map can be found in many places of the code and effectively represents an ID of ECU, just thinking if it makes sense to define corresponding types, e.g. `class ECU { class/using/typedef ID .....` so it can be referenced as Uptane::ECU::ID ...

Reply:
Not a bad thought. I'll look into it.